packages/lesswrong/lib/components.ts

```diff
@@ -238,7 +238,9 @@ importComponent("SuggestAlignment", () => require('../components/posts/SuggestAl
 importComponent("Pingback", () => require('../components/posts/Pingback'));
 importComponent("PingbacksList", () => require('../components/posts/PingbacksList'));
 importComponent("PostsItemMeta", () => require('../components/posts/PostsItemMeta'));
-importComponent("PostsItem2", () => require('../components/posts/PostsItem2.tsx'));
+importComponent("PostsItem", () => require('../components/posts/PostsItem.tsx'));
+importComponent("BookPostsItem", () => require('../components/posts/BookPostsItem.tsx'));
+importComponent("FriendlyPostsItem", () => require('../components/posts/FriendlyPostsItem.tsx'));
```

Comment:
Seeing as we are now not (??) doing the book/friendly distinction I think it would be clearer in the long run to name these `LWPostsItem` and `EAPostsItem`

Reply:
Done!